proteus/TriangleTools.py

```diff
@@ -11,10 +11,59 @@
 import triangleWrappers
 import TriangleUtils
 import TriangleFileUtils
+import MeshTools
+
```

Comment:
This looks good. I would put it in MeshTools.py though. TriangleTools  was supposed to be specifically for `triangle`.  We could add a TetgenTools module and start pulling out a lot of the stuff around tetgen calls (e.g. from NumericalSolution). Anyway, not a big deal right now. I'd just move it to MeshTools